addons/point_of_sale/models/res_company.py

```diff
@@ -12,11 +12,19 @@ class ResCompany(models.Model):
             ], default='closing', string="Update quantities in stock",
             help="At the session closing: A picking is created for the entire session when it's closed\n In real time: Each order sent to the server create its own picking")
     point_of_sale_use_ticket_qr_code = fields.Boolean(
-        string='Use QR code on ticket',
-        help="Add a QR code on the ticket, which the user can scan to request the invoice linked to its order.")
+        string='Self-service invoicing',
+        help="Add the self-service invoicing, which the user can request the invoice linked to its order by himself.")
     point_of_sale_ticket_unique_code = fields.Boolean(
         string='Generate a code on ticket',
         help="Add a 5-digit code on the receipt to allow the user to request the invoice for an order on the portal.")
+    point_of_sale_ticket_portal_url_display_mode = fields.Selection([
+            ('qr_code', 'QR code'),
+            ('url', 'URL'),
+            ('qr_code_and_url', 'QR code + URL'),
+        ], default='qr_code',
+        string='Way of print',
+        help="Choose how the url to the portal will be display.",
```

Comment:
please discuss with @dama-odoo about this `help` text

Reply:
Hi guys ! As discussed, I see this has been updated correctly 👌
![image](https://github.com/odoo/odoo/assets/88381670/494febae-706b-4547-b048-4d282b05ee0d)
Though, about the help message (we didn't discussed), I'd suggest:
> Choose how the URL to the portal will be printed on the receipt.